addon/components/frost-info-bar.js

```diff
@@ -5,5 +5,6 @@ export default Component.extend({
   // == Properties ============================================================
 
   classNames: ['frost-info-bar'],
-  layout
+  layout,
+  hook: 'info-bar'
```

Comment:
The only place a default hook might be ok would be singletons. Things that cannot exist more than once in the DOM.